website/src/components/Navbar.vue

```diff
@@ -1,16 +1,18 @@
 <template>
   <chakra.nav
     h="60px"
-    w="100vw"
+    w="100%"
     px="4"
     d="flex"
     position="fixed"
     align-items="center"
     shadow="sm"
     bg="white"
     z-index="10"
+    border-top="6px solid"
+    border-color="vue.500"
   >
-    <CFlex align-items="center" flex="1" />
+    <CFlex align-items="center" :flex="{ base: 0, sm: 1 }" />
```

Comment:
I think I might have merged a bug with the new CSS properties in the pull request.

Gonna open a PR fixing this.